api/CompositionEvent.json

```diff
@@ -104,10 +104,10 @@
           "spec_url": "https://w3c.github.io/uievents/#dom-compositionevent-data",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "43"
```

Comment:
As odd as this "43" may seem, this is one of the few features that actually _was_ added in Chrome 43 -- see https://source.chromium.org/chromium/chromium/src/+/36bee4e52d58e0291961877c781c4dfdeb602299